addons/website_blog/views/snippets/s_blog_posts.xml

```diff
@@ -16,8 +16,8 @@
         <a class="s_blog_posts_post_cover flex-grow-0 flex-shrink-0 align-self-baseline position-relative" t-att-href="data['call_to_action_url']" t-att-title="'Read' + data['name']">
```

Comment:
Commit message:
- `Some of the layout are working fine` => "layouts" or "some layouts"
- `the problem gets worst` => worse
- `(you can just download the one on the PR)` => what image are you talking about ?
- `it will get even worst.` => worse
- `introduced with 3c0d98b` => nitpick but missing period at the end of the sentence

Reply:
Whoops, fixed.
Note that for the last part about the missing period, I did not add it on purpose since the link is on its own line and going "off screen" (over the 72 char).
Also note that it's the only case where I don't use markdown, when the commit is the last part of the sentence and that the link has to be added on a new line (because previous line is already close to 72 char). If it was in the middle of the sentence (at 40 char eg), I would have insert it through markdown

Reply:
Aah I didn't notice you added it like that, I only saw the "rendering" so I thought there was a ref somewhere (I wouldn't have said that if had seen it) :open_mouth: